src/ui/image/Image.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect, useRef } from "react"
+
+import { Loader, Button } from "ui"
+
+import csx from "./Image.scss"
+
+namespace Image {
+    export interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
+        className?: string;
+        lowQuality: string;
+        fallback?: string;
+    }
+}
+
+const Image = ({ lowQuality, fallback, src, className, width, height, ...props }: Image.Props) => {
+    const [url, setUrl] = useState(lowQuality || src)
+    const [error, setError] = useState(false)
+
+    useEffect(() => {
+        if (url === lowQuality) loadImage();
+    }, [])
+
+    const loadImage = (): void => {
+        setError(false)
+        fetch(src).then(r => r.blob()).then(b => {
+            const fileReader = new FileReader();
+            fileReader.onload = function () {
+                setUrl(this.result + "");
+            };
+            fileReader.readAsDataURL(b);
+        })
+    }
+
+    const onErrorHandler = (): void => {
```

Comment:
rename to handleError and useCallback here

Reply:
done